### DISCUSSION TITLE
Are migration guides available?

### DISCUSSION THREAD

**Discussion by buzoherbert** [ExpediaGroup/graphql-kotlin#1399] [General] [answered]
I am not sure if this is the right repository, please direct me to the right one if this is not the one.

I am trying to upgrade com.expediagroup:graphql-kotlin-spring-server from version 3.6.7 to version 5.3.2. I have found that some classes are not available anymore on the package. (ExecutionPath , SimpleKotlinGraphQLError) and I am not sure how to replace them.

Is there a migration guide for the different upgrades to the package?

Thanks a lot in advance!

**Comment by dariuszkuc** [comment, marked as answer]
Hello 👋 
Apologies but we currently don't have any migration guides. Breaking changes are listed in the release notes.

We did a big namespace cleanup in v4.0 -> https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/4.0.0

**Comment by buzoherbert** [comment]
Thanks a lot. This is very helpful!
